### ISSUE TITLE
change the NavigationDrawerLayout and add listener 

### ISSUE THREAD

**Issue by smartshuai** [DenisMondon/material-design-library#58]
hi , thanks  for your open source code.
If I want to change the String in the bottom of NavigationDrawerLayout ,change the string"help and feedback" to "about us", and I want to go to another activity or fragment when I click this item, how should I do ? 
Thanks.

**Comment by 28492** [comment]
Yes, you can. I don't know, if library in depecies you can modify, than you must download this library and include to your project.
Easily way is modify settings button
Than:
Res\values\strings
From


<string name="mdl_settings">Settings</string>

To

<string name="mdl_settings">About us</string>


And icon res\drawable ic_settings

Smarter way is added your method to NavigationDrawerBottomHandler and in 
NavigationDrawerListItemBottom add to constructor in switch after default:

setTitle(context, R.string.your_identific_in_strings);
 setIcon(context, R.drawable.ic_help);
